Projects/NET/NET04/NET04-AWS100.md

```diff
@@ -1,28 +1,31 @@
-# NET04-AWS100 — Create three Billing Alarms
+# NET04-AWS100 — Host a simple static webpage with S3 and CloudFront
```

Comment:
- "simple" can be removed IMO. Title is already pretty long, will take up a lot of space in the table view.

Reply:
Right, we can specify in the description. I just would not want people to think they need to work on a whole website.

Reply:
Welp, I just noticed that it line-breaks in the table view so it's not actually an issue.

Hm, good point. yeah @madebygps what do you think?